### ISSUE TITLE
Add support for Node.js

### ISSUE THREAD

**Issue by jordanbtucker** [dullin/highlightjs-vba#1]
Please add support for Node.js. It currently only works in the browser.

**Comment by tim-elmer** [comment]
Not sure if this is still relevant - or if you're experiencing the exact same issue I am (using this in Antora), but I managed to get it to work by changing the module to use an export:

```vba
module.exports = function(hljs) {
```

**Comment by jordanbtucker** [comment]
Yes, the problem is that `src/vba.js` is an ECMAScript module but the `type` field is not set to `module` in `package.json`.

See https://nodejs.org/dist/latest-v17.x/docs/api/esm.html#enabling
